modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -127,7 +127,20 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
                     {{paramName}};
                 {{/isDate}}
                 {{^isDate}}
+                {{#isExplode}}
+                {{#isPrimitiveType}}
                 localVarQueryParameter['{{baseName}}'] = {{paramName}};
+                {{/isPrimitiveType}}
+                {{^isPrimitiveType}}
+                let param: keyof typeof {{baseName}};
+                for (param in {{baseName}}) {
```

Comment:
how about just
```suggestion
                for (let param of Object.keys({{baseName}})) {
```
to avoid enumerating prototype properties?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in?retiredLocale=de#iterating_over_own_properties_only